pkg/controller/deployment/deployment_controller.go

```diff
@@ -482,7 +482,7 @@ func (dc *DeploymentController) handleErr(err error, key interface{}) {
 	}
 
 	if dc.queue.NumRequeues(key) < maxRetries {
-		klog.V(2).Infof("Error syncing deployment %v: %v", key, err)
+		klog.V(2).InfoS("Error syncing deployment", "deployment", key, "err", err)
```

Comment:
Please use cache.SplitMetaNamespaceKey(key) and use klog.KRef

Reply:
key is already in `ns/name` format, right? do we need to split and join again?